### PR TITLE
Update coordinate benchmarks section

### LINKED ISSUE #15: Table 1 as matrix
@perrygreenfield commented:

Wouldn't the table be better represented as a matrices? Or perhaps as a matrix as a crude image with colors or intensities representing the size of the quantity. Is the difference between median and mean significant enough to warrant showing both? A link to the actual numbers may satisfy readers that want the exact values, but I suspect the majority don't need to see that precision.

### PR DESCRIPTION
**Pull request by adrn**
I removed Table 1 and simplified the visualization to remove the lower triangle. I also added a colorbar:

image

What do you think about the colorbar label?

Fixes #15

### CODE REVIEW

File: main.tex [hunk 1]
```diff
@@ -613,59 +613,24 @@ \subsubsection{Accuracy of coordinate transformations}
...
-\caption{Comparison of the accuracy of the FK4 to Galactic transformation between different packages.\label{tab:coordinate_benchmarks}}
-\end{table}
+\figurename~\ref{fig:coordinate_benchmarks} visualizes the relative accuracy of
+the conversion from FK4 to Galactic coordinates for all pairs of tools.
+This figure shows, for example, that \astropy, Kapteyn, and PyTPM agree
+perfectly (light colors, small differences), while PALpy, pySLALIB, and PyAST
+also agree perfectly amongst themselves.
+However, there is an offset of around 0.2\arcsec\space with the former three
+packages.
+Finally, PyEphem disagrees with other packages by 0.4--0.8\arcsec.
```
Comment (bsipocz) [inline main.tex]: "disagrees with **all** the other packages"?

**Comment by pllim** [comment]
Should we make "astropy" bold on the plot axes so eyes are drawn to it first? ~~And without a caption here, darker color means better or worse?~~ (Nevermind, I was so enticed by the diagram here I forgot to look at the diff.)

**Comment by bsipocz** [comment]
What about removing the mention of the packages that are not inlcuded in the figure from the list above and just say, among others?

**Comment by bsipocz** [comment]
(oh, sorry, it must have been a misclick...)

**Pull request merged**